server.js

```diff
@@ -18,9 +18,66 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send("Go to /thoughts to see the data added to this endpoint.");
```

Comment:
Great that you added the endpoint here for those who don't know what to search for 😄